plugins/modules/cloudflare_dns.py

```diff
@@ -749,6 +774,22 @@ def ensure_dns_record(self, **kwargs):
             }
             search_value = str(params['cert_usage']) + '\t' + str(params['selector']) + '\t' + str(params['hash_type']) + '\t' + params['value']
 
+        if params['type'] == 'CAA':
+            caa_data = {
+                "flags": 0,
```

Comment:
Why not make the flags configurable as well?

Reply:
sry I could have provided more inside on the Flags
RFC 8659 describes the Flags > https://www.rfc-editor.org/rfc/rfc8659#name-mechanism
But cloudflare did not implement the critical flags, and it is only possible to send non-critical records at this point in time